backend/src/git.rs

```diff
@@ -335,6 +379,21 @@ impl Interface {
         Ok(())
     }
 
+    /// Pushes the commits to a branch instead of the master branch.
```

Comment:
I didn't want to update git push because I wasn't sure about it messing with anything else. I can look into something something wildcard refspecs later when I get done.